### PR TITLE
Get rid of deprecation warnings in tests and examples

### PR DESCRIPTION
**Pull request by AndKaminer**
I changed the methods from the camel case variants to snake case.

### CODE REVIEW

File: examples/human.py [hunk 1]
```diff
@@ -73,20 +73,20 @@ def userConsole(args):
...
                 print("\t" + str(actionStr))
 
         elif (userInputStr == "valid"):
             print("Valid action-object combinations:")
-            print(env.getValidActionObjectCombinationsWithTemplates())
+            print(env.get_valid_action_object_combinations_with_templates())
 
         elif (userInputStr == 'goals'):
-            print(env.getGoalProgressStr())
+            print(env.get_goal_progress_srt())
```
Comment (MarcCote) [inline examples/human.py:89]: There's a typo here `..._srt` vs. `..._str`. But fundamentally, do we need it in the first place? I'd be in favor to drop the `_str` all together. The docstring should be clear enough.

**Review by MarcCote — CHANGES_REQUESTED** [review]

File: examples/human.py (hunk 1 above)
Comment (AndKaminer) [reply examples/human.py:89]: Oh oops. Somehow that ran on my machine, even with the typo. I'm fine with that. My policy was to just keep the method the calls exactly the same, but I'm happy to change.

Comment (AndKaminer) [reply examples/human.py]: Done

Comment (MarcCote) [reply examples/human.py]: :+1: Just make sure to make the change in scienceworld.py too.

Re: testing, in order to trigger this, you need to run the script and type "goals" as one of the in-game actions.

Comment (AndKaminer) [reply examples/human.py]: As far as I can tell, get_goal_progress_str is the only method where removing the end type doesn't get rid of any important information, so I think w get rid of just that one.

**Review by MarcCote — APPROVED** [review]

**Pull request merged**